toolchain/sem_ir/inst_namer.cpp

```diff
@@ -705,6 +705,10 @@ auto InstNamer::CollectNamesInBlock(ScopeId top_scope_id,
         add_inst_name(out.TakeStr());
         continue;
       }
+      case ImportCppDecl::Kind: {
+        add_inst_name("import_cpp");
```

Comment:
```suggestion
        add_inst_name("Cpp.import_cpp");
```

We're typically trying to reflect the identifier as part of instruction names so that the parallels with code are easier to see. How about this, which is close to the `ImportDecl` format?

Reply:
Done.